lib/tools/broccoli/node-modules-dest-copy.ts

```diff
@@ -75,6 +75,11 @@ export class DestCopy implements IBroccoliPlugin {
 		if(isPlugin) {
 			this.$pluginsService.prepare(dependency).wait();
 		}
+
+		if (dependency.name == constants.TNS_CORE_MODULES_NAME) {
+			shelljs.cp("-Rf", path.join(this.outputRoot, constants.TNS_CORE_MODULES_NAME, "*"), this.outputRoot);
+			this.$fs.deleteDirectory(path.join(this.outputRoot, constants.TNS_CORE_MODULES_NAME)).wait();
```

Comment:
Use can extract `path.join(this.outputRoot, constants.TNS_CORE_MODULES_NAME)` to separate variable